### ISSUE TITLE
Two factions

### ISSUE THREAD

**Issue by Dzierzan** [OpenHV/OpenHV#116]
By default Hard Vacuum doesn't have enough assets to have two different looking factions. There are many indeed units/tanks, but IMO that is not enough. Since Tyrian's assets are free to use as well, I suggest using some of its assets, repaint if needed and create 2nd faction. At first, it would be good enough if both factions were symmetrical with just different looking units, but in the long run, it would have been perfect if they were assymetrical.

Here are some examples:

**Radar** (the orb at the bottom could be removed)
![unknown (2)](https://user-images.githubusercontent.com/17529329/87705725-a2e68d00-c79e-11ea-81f2-3de8597f3621.png)

**Base/Conyard**
![unknown](https://user-images.githubusercontent.com/17529329/87705758-ad088b80-c79e-11ea-81a2-6166a6037d8d.png)

**Turret** (the base could be used from regular HV turret base)
![unknown (1)](https://user-images.githubusercontent.com/17529329/87705802-bc87d480-c79e-11ea-9f11-efc1c1a068cc.png)

**Factory** from which units go up instead of classic down direction?
![unknown (3)](https://user-images.githubusercontent.com/17529329/87705834-cdd0e100-c79e-11ea-9524-42edcd22cd5c.png)

Basically 2nd faction could have more darkish tone (hey, even my voice would fit them).

**Comment by GraionDilach** [comment]
I previously already suggested using the Tyrian effects at one point to spice up the GFX. There are some stuff which looks odd currently (like the fact tanks lack an explosion).
